internal/page/attorney/mock_test.go

```diff
@@ -41,3 +42,11 @@ var (
 		ActorType:   actor.TypeReplacementTrustCorporation,
 	}
 )
+
+func evalT[T any](fn func(*testing.T) T, t *testing.T) T {
```

Comment:
Cool! Now we're using funcs more in test tables this will come in handy. Could we make the name slightly more verbose for future maintainers? Something like `evalMock()` although that does signal its limited to mocks which may not be the case in every test...maybe the name is fine, I dunno 🤷

Reply:
I was also struggling to name this thing. If we come up with anything later it will be easy to rename